src/lib/experiments/tests/ads-in-merch.ts

```diff
@@ -1,7 +1,7 @@
 import type { ABTest } from '@guardian/ab-core';
 
-export const billboardsInMerchHigh: ABTest = {
-	id: 'BillboardsInMerchHigh',
+export const adsInMerch: ABTest = {
+	id: 'AdsInMerch',
 	author: '@commercial-dev',
 	start: '2022-12-07',
 	expiry: '2023-11-30',
```

Comment:
Are we happy with this for the expiry?

Reply:
I think this is okay. I fully intend to tear out all the AB test stuff before we go live, as we won't actually be running an AB test for Black Friday. It'll just be how merchandising slots work going forward.